dynamic_rest/serializers.py

```diff
@@ -213,11 +213,20 @@ def __init__(
         super(WithDynamicSerializerMixin, self).__init__(**kwargs)
 
         self.envelope = envelope
-        self.sideloading = sideloading
         self.debug = debug
         self.dynamic = dynamic
         self.request_fields = request_fields or {}
-        self.embed = embed if sideloading is None else not sideloading
+
+        # sideloading modifies top-level response keys,
+        # so it requires an envelope
+        if envelope is False:
```

Comment:
My initial approach here was fail-soft by overriding, e.g. similar to the way `include_fields='*'` overrides `exclude_fields`, rather than fail-hard by throwing an exception... but I'm open to being more strict